core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -205,6 +204,8 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal
     tws.map(_ => Nil)(akka.dispatch.ExecutionContexts.sameThreadExecutionContext)
   }
 
+  protected def generateUUID(pr: PersistentRepr): UUID = UUIDs.timeBased()
```

Comment:
I'd add a comment so someone doesn't inline this :)

Reply:
I added a link to the original issue - hope this is ok